client/service/commands.go

```diff
@@ -1,93 +1,241 @@
 package service
 
 import (
-	"encoding/json"
 	"errors"
 	"fmt"
+	"html"
 	"log"
 	"net/http"
+	"net/url"
+	"regexp"
+	"strings"
 	"time"
 
 	"github.com/databrickslabs/databricks-terraform/client/model"
 )
 
+var (
+	outRE          = regexp.MustCompile(`Out\[[\d\s]+\]:\s`)
+	tagRE          = regexp.MustCompile(`<[^>]*>`)
+	exceptionRE    = regexp.MustCompile(`.*Exception: (.*)`)
+	errorMessageRE = regexp.MustCompile(`ErrorMessage=(.*)\n`)
```

Comment:
Can we add a comments to this to understand when each Regex are being used, the first, second and last are a bit unclear.

Reply:
added